packages/relay/src/utils.ts

```diff
@@ -65,4 +65,44 @@ export class Utils {
   public static generateRequestId = (): string => {
     return crypto.randomUUID();
   };
+
+  /**
+   * Estimates the total fee in tinybars for file transactions based on the given call data size,
+   * file chunk size, and the current network exchange rate.
+   * Estimates the transaction fees for file create and file append transactions based on the provided
```

Comment:
Written twice

Reply:
Ah good catch I wonder why it's duplicated. Might've missed it during resolving conflicts. Updated.

Reply:
Ah good catch I wonder why it's duplicated. Might've missed it during resolving conflicts. Updated.